wal/decoder.go

```diff
@@ -73,6 +73,13 @@ func (d *decoder) decodeRecord(rec *walpb.Record) error {
 		return err
 	}
 
+	// handle padding bytes
```

Comment:
can we document in more detail? e.g. I want to understand where we get that `56`. Thanks.